docs/tutorial/external_dependencies.rst

```diff
@@ -12,33 +12,42 @@ To accomplish this BLT provides a ``DEPENDS_ON`` option for the
 ``blt_add_library()`` and ``blt_add_executable()`` macros that supports both CMake targets 
 and external dependencies registered using the ``blt_register_library()`` macro.
 
-The ``blt_register_library()`` macro allows you to reuse all information needed
+The ``blt_import_library()`` macro allows you to reuse all information needed
 for an external dependency under a single name.  This includes any include
 directories, libraries, compile flags, link flags, defines, etc.  You can also
 hide any warnings created by their headers by setting the
 ``TREAT_INCLUDES_AS_SYSTEM`` argument.
 
-For example, to find and register the external dependency *axom* as a BLT registered library, you can simply use:
+For example, to find and register the external dependency *axom* as a CMake target, you can simply use:
 
 .. code-block:: cmake
 
     # FindAxom.cmake takes in AXOM_DIR, which is a installed Axom build and
     # sets variables AXOM_INCLUDES, AXOM_LIBRARIES
     include(FindAxom.cmake)
-    blt_register_library(NAME      axom
-                         TREAT_INCLUDES_AS_SYSTEM ON
-                         DEFINES   HAVE_AXOM=1
-                         INCLUDES  ${AXOM_INCLUDES}
-                         LIBRARIES ${AXOM_LIBRARIES})
+    blt_import_library(NAME      axom
+                       TREAT_INCLUDES_AS_SYSTEM ON
+                       DEFINES   HAVE_AXOM=1
+                       INCLUDES  ${AXOM_INCLUDES}
+                       LIBRARIES ${AXOM_LIBRARIES})
 
 Then *axom* is available to be used in the DEPENDS_ON list in the following
-``blt_add_executable()`` or ``blt_add_library()`` calls.
+``blt_add_executable()`` or ``blt_add_library()`` calls, or in any CMake command that accepts a target.
+CMake targets created by ``blt_import_library()`` are ``INTERFACE`` libraries that can be installed
```

Comment:
> CMake targets created by ``blt_import_library()`` are ``INTERFACE`` libraries that can be installed and exported

In that case, would it make sense for the new macro to be `blt_interface_library` instead of `blt_import_library`?
Specifically, are there different semantics in CMake for import libraries vs. interface libraries? And does the new macro better match one of these?

Reply:
I view "import" as a verb not an adjective but the word is pretty overloaded in CMake.  Maybe clarify the docs with "are CMake `INTERFACE` libraries"

Reply:
Thanks @white238 -- to clarify, I'm happy with `blt_import_library` as the name as long as it is not unintentionally implying something CMake-related about the semantics of the generated targets.